src/apps/qe_to_phoebe_utils.cpp

```diff
@@ -353,6 +355,12 @@ void ElPhQeToPhoebeApp::testBackTransform(
       Eigen::VectorXcd polar = couplingElPh.polarCorrectionPart1(q3C, eigenVector3);
       polarData.push_back(polar);
 
+
+      couplingElPh.calcCouplingSquared(eigenVector1, eigenVectors2,
```

Comment:
it appears this is called twice, once here and once below -- seems like that might not make sense?